docs/NUMPY_COMPARISON.md

```diff
@@ -297,12 +297,13 @@ This is a collection of common exercises using numpy, and their equivalent solut
 ```
 
 ```v
->>> import vtl.num
+>>> import vtl.num as np
```

Comment:
why do we want to do this instead of using it as `num.{function_name}`?

Reply:
Only to align it with the whole context and the title of the example: "1. Import the numpy package under the name np"